app/code/Magento/CmsGraphQl/composer.json

```diff
@@ -5,6 +5,7 @@
     "require": {
         "php": "~7.1.3||~7.2.0||~7.3.0",
         "magento/framework": "*",
+        "magento/module-store": "*",
```

Comment:
```suggestion
        "magento/module-store": "*",
```

Reply:
??? I don't see any changes or comment @lenaorobei

Reply:
`magento/module-store` is redundant dependency

Reply:
If I don't set it, I get the error: Module Magento\CmsGraphQl has undeclared dependencies: hard [Magento\Store]